### PR TITLE
Story1

### PR DESCRIPTION
**Pull request by Sen-i**
Final review!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+	
+	<head>
+		<title>PORTFOLIO</title>
+			<link rel="stylesheet" type="text/css" href="portfolio.css">
```
Comment (soneillcode) [inline index.html]: indentation here is incorrect, should be inline with the element above.

File: portfolio.css [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+		background-image: linear-gradient(to top, rgba(73,23,166,0), rgba(73,23,166,1));
+		padding:5px;
+}
+footer{
+		width:100%;
+		background-color:#868687;
+}
+.portcomment{
+	padding:40px;
+}
```
Comment (soneillcode) [inline portfolio.css]: needs a new line at the end of the file

File: index.html [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+		<div class="contactme" id="contactme">
+				<h2>Contact Me</h2>
+			<div class="aboutmecontainer">
+					<div>
+					<h3>Is this the dev you are looking for?</h3>
+					<p>If so mail me</p>
+					<a href="mailto:vinc_944@hotmail.com" class="email">EMAIL: vinc_944@hotmail.com</a>
+					</div>
+				</div>
+			<div class="social">
```
Comment (soneillcode) [inline index.html:78]: these links should link to your actual profiles.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+	<head>
+		<title>PORTFOLIO</title>
+			<link rel="stylesheet" type="text/css" href="portfolio.css">
+			<link rel="stylesheet" type="text/css" href="normalize.css">
+			<meta name="viewport" content="width=device-width, initial-scale=1">
+	</head>
+	<body>
+	<nav class="sticky">
+	 	<nav class="navholder">
+			<!--<div class="logo"><a href="#"><img class="logoimg" alt="" src=></a></div>-->
```
Comment (soneillcode) [inline index.html]: remove commented out code.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+<html lang="en">
+	
+	<head>
+		<title>PORTFOLIO</title>
+			<link rel="stylesheet" type="text/css" href="portfolio.css">
+			<link rel="stylesheet" type="text/css" href="normalize.css">
+			<meta name="viewport" content="width=device-width, initial-scale=1">
+	</head>
+	<body>
+	<nav class="sticky">
```
Comment (soneillcode) [inline index.html]: should only be one nav, use div for the not nav.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+	
+	<head>
+		<title>PORTFOLIO</title>
+			<link rel="stylesheet" type="text/css" href="portfolio.css">
+			<link rel="stylesheet" type="text/css" href="normalize.css">
```
Comment (soneillcode) [inline index.html]: order matters, this means that normalize.css be linked first.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+			<!--<div class="logo"><a href="#"><img class="logoimg" alt="" src=></a></div>-->
+			<div class="contact"><a href="#contactme">CONTACT ME</a></div>
+			<div class="portfolio"><a href="#portfolio">PORTFOLIO</a></div>
+			<div class="about"><a href="#aboutme">ABOUT ME</a></div>
+			<div class="home"><a href="#">HOME</a></div>
+		</nav>
+		</nav>
+		<div class="introbox">
+			<div>
+			<h1>Hello There</h1>
```
Comment (soneillcode) [inline index.html]: indentation.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+			<h3>Welcome to my world!</h3>
+			<p>A short description of my world as a nation can be given here. It's a world with only slightly higher tech, due to the general resources scarcity of the place. There are many land-based colonies, like a few settlements in the southeastern half. Like most. Every single one of these is too small to be considered a nation, but they are still human sized. And they're all just child to me.The geography is mountainous, but there's also a lot of forests, thanks to the many mines spread out around the country. I have also personally researched the science of forest, and it's the second to best one.And this is the old map, and the one I've gotten myself from</p>
```
Comment (soneillcode) [inline index.html]: lines have a max length, you can wrap text in the file and HTML will still render it correctly.

File: portfolio.css [hunk 9]
```diff
@@ -0,0 +1,289 @@
...
+	font-family: 'Nanum Gothic', sans-serif;
+}
+@import url('https://fonts.googleapis.com/css?family=Nanum+Gothic&display=swap');
+
+a{
+	text-decoration: none;
+	text-decoration-color: #736d6c;
+	color:inherit;
+}
+.navholder{
```
Comment (soneillcode) [inline portfolio.css]: remove empty class

File: portfolio.css [hunk 10]
```diff
@@ -0,0 +1,289 @@
...
+	
+}	
+/*@media only screen and (max-width:850px)
```
Comment (soneillcode) [inline portfolio.css]: remove commented out css

**Review by soneillcode — CHANGES_REQUESTED** [review]

File: index.html [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+					 one I've gotten myself from
+				</p>
+			</div>
+		</div>
+		<h2 id="aboutme">ABOUT ME</h2>
+		<div class="aboutmecontainer">
+			<div class="imgcontainer">
+				<img class ="abmimg "src="Assets/panda.jpg" alt="about me img">
+			</div>
+				<div class="contentcontainer">
```
Comment (soneillcode) [inline index.html]: should be inline with other sibling

File: index.html [hunk 12]
```diff
@@ -0,0 +1,106 @@
...
+				</div>
+		</div>
+		<h2 id="portfolio">Portfolio</h2>
+		<div class="portcontainer">
+			<div class="rowcontainer">
+				<div class="columncontainer">
+					<div class="imgcontainer">
+						<img class ="portimg "src="Assets/lights.jpeg" alt="about me img">
+					</div>
+						<div class="portcontainer1">
```
Comment (soneillcode) [inline index.html]: should be inline

File: index.html [hunk 13]
```diff
@@ -0,0 +1,106 @@
...
+							<h3 class="porttitle">Placeholder</h3>
+							<p class="portcomment">It’s time to reveal the face of the magician behind the curtains. </p>
+						</div>
+				</div>
+				<div class="columncontainer">
+					<div class="imgcontainer">
+						<img class ="portimg "src="Assets/lights.jpeg" alt="about me img">
+					</div>
+						<div class="portcontainer1">
```
Comment (soneillcode) [inline index.html]: should be inline

File: portfolio.css [hunk 14]
```diff
@@ -0,0 +1,271 @@
+*{
+	box-sizing:border-box;
```
Comment (soneillcode) [inline portfolio.css]: https://google.github.io/styleguide/htmlcssguide.html worth reading

File: portfolio.css [hunk 15]
```diff
@@ -0,0 +1,271 @@
...
+	border-color:transparent;
+	float:right;
+}
+.home:hover{
+	border-bottom:2px solid;
+	border-color:#2bd128;
+}
+@mediaonlyscreenand(max-width:850px){
+	.home{
+	background-image:url(Assets/home.png);
```
Comment (soneillcode) [inline portfolio.css]: indentation here, line  60 - 66

File: portfolio.css [hunk 16]
```diff
@@ -0,0 +1,271 @@
...
+	background-color:#464646;
+	display: flex;
+	align-items:center;
+	justify-content:flex-start;
+	font-size:1.5rem;
+	color: #feda4a;
+}
+nav{
+	font-size:1.5rem;
+	background-color:transparent;
```
Comment (soneillcode) [inline portfolio.css]: display: none;

File: index.html [hunk 17]
```diff
@@ -0,0 +1,95 @@
...
+					<img src="Assets/twitter.png" alt="twitter" class="icon">
+				</a>
+				<a href="https://www.reddit.com/user/h3x4r7">
+					<img src="Assets/redimg.png" alt="reddit" class="icon">
+				</a>
+			</div>
+		</div>
+		<footer>&copy; Vincent 2020</footer>
+	</body>
+</html>
```
Comment (soneillcode) [inline index.html]: new line at the end of files

File: portfolio.css [hunk 18]
```diff
@@ -0,0 +1,264 @@
...
+	padding: 0;
+	background-image: url(Assets/worldlights.jpg);
+	background-repeat: no-repeat;
+	flex-direction: column;
+	background-attachment: fixed;
+	background-size: cover;
+	background-color: #464646;
+	display: flex;
+	align-items: center;
+	font-size: 1.5rem;
```
Comment (soneillcode) [inline portfolio.css]: This should be a percentage so rem will work

File: portfolio.css [hunk 19]
```diff
@@ -0,0 +1,264 @@
...
+	border-color: #2bd128;
+}
+@media only screen and (max-width: 905px) {
+	.home {
+		background-image: url(Assets/home.png);
+		background-size: cover;
+		height: 50px;
+		width: 50px;
+		padding: 0px;
+		float: right;
```
Comment (soneillcode) [inline portfolio.css]: This CSS is repeated several times. When you see repeating CSS, it's a good idea to create a generic class for the common properties.

**Review by soneillcode — APPROVED** [review]
All good!

File: index.html [hunk 20]
```diff
@@ -0,0 +1,95 @@
...
+				<div class="about navbutton"><a class="abouticon"  href="#aboutme">ABOUT ME</a></div>
+				<div class="home navbutton"><a class="homeicon" href="#home">HOME</a></div>
+			</nav>
+		</div>
+		<div class="introbox">
+			<div>
+				<h1 id="home">Hello There</h1>
+				<h3>Welcome to my world!</h3>
+				<p>This is my world created with html and css.
+				</p>
```
Comment (mporam) [inline index.html:23]: closing tag doesnt need to be on a new line

File: index.html [hunk 21]
```diff
@@ -0,0 +1,95 @@
...
+			</div>
+		</div>
+		<h2 id="aboutme">ABOUT ME</h2>
+		<div class="aboutmecontainer">
+			<div class="imgcontainer">
+				<img class ="abmimg " src="Assets/panda.jpg" alt="about me img">
```
Comment (mporam) [inline index.html:29]: img is not a word. alt text needs to be English so screen readers can read it out

File: index.html [hunk 22]
```diff
@@ -0,0 +1,95 @@
...
+			</div>
+			<div class="contentcontainer">
+				<h3>Vincent Pires</h3>
+				<p>Hi I'm Vincent, an aspiring full stack developer currently training at Mayden Academy.
+				</p>
```
Comment (mporam) [inline index.html:34]: again, doesnt need its own line

File: index.html [hunk 23]
```diff
@@ -0,0 +1,95 @@
...
+				<a href="https://github.com/Sen-i">
+					<img src="Assets/github.png" alt="github" class="icon">
+				</a>
+				<a href="https://www.facebook.com/Vincdaman">
+					<img src="Assets/fbimg.png" alt="facebook" class="icon">
+				</a>
+				<a href="https://twitter.com/vincdaman">
+					<img src="Assets/twitter.png" alt="twitter" class="icon">
+				</a>
+				<a href="https://www.reddit.com/user/h3x4r7">
```
Comment (mporam) [inline index.html:88]: sure you want employers seeing your reddit??

File: portfolio.css [hunk 24]
```diff
@@ -0,0 +1,257 @@
+* {
+	box-sizing: border-box;
+	font-family: 'Nanum Gothic',sans-serif;
```
Comment (mporam) [inline portfolio.css:3]: should set font-family on the html tag, not on `*`

File: portfolio.css [hunk 25]
```diff
@@ -0,0 +1,257 @@
...
+html {
+	font-size: 62.75%;
+}
+a {
+	text-decoration: none;
+	color: inherit;
+}
+.sticky {
+	width: 100%;
+	background-color:#000000;
```
Comment (mporam) [inline portfolio.css:14]: missing space after `:`

File: portfolio.css [hunk 26]
```diff
@@ -0,0 +1,257 @@
...
+	font-family: 'Nanum Gothic',sans-serif;
+}
+html {
+	font-size: 62.75%;
+}
+a {
+	text-decoration: none;
+	color: inherit;
+}
+.sticky {
```
Comment (mporam) [inline portfolio.css:12]: class names should describe what something is, not what it looks like. this seems to do neither? its not sticky, or a stick?

File: portfolio.css [hunk 27]
```diff
@@ -0,0 +1,257 @@
...
+	width: 100%;
+	background-color:#000000;
+}
```
Comment (mporam) [inline portfolio.css:15]: css syntax is:

```
selector {

}

selector {

}
```

your missing the line breaks between each selector

File: portfolio.css [hunk 28]
```diff
@@ -0,0 +1,257 @@
...
+@media only screen and (max-width: 905px) {
+	.home {
+		background-image: url(Assets/home.png);
+		background-size: cover;
+	}
+	.navbutton {
+		height: 50px;
+		width: 50px;
+		padding: 0px;
```
Comment (mporam) [inline portfolio.css:57]: dont need px on 0 values

File: portfolio.css [hunk 29]
```diff
@@ -0,0 +1,257 @@
...
+		float: right;
+	}
+	.homeicon {
+		display:none;
```
Comment (mporam) [inline portfolio.css:61]: missing space after `:`

File: portfolio.css [hunk 30]
```diff
@@ -0,0 +1,257 @@
...
+	}
+	.about {
+		background-image: url(Assets/aboutme.png);
+		background-size: cover;
+	}
+	.abouticon {
+		display:none;
```
Comment (mporam) [inline portfolio.css:68]: missing space again

File: portfolio.css [hunk 31]
```diff
@@ -0,0 +1,257 @@
...
+	nav {
+		width: 100%;
+	}
+}
+.portfolio {
+	margin-left: 15px;
+	padding: 10px;
+	border-bottom: 2px solid;
+	float: right;
+	border-color: transparent;
```
Comment (mporam) [inline portfolio.css:93]: why not just remove the border?

Comment (mporam) [reply portfolio.css:93]: ooo i see

File: portfolio.css [hunk 32]
```diff
@@ -0,0 +1,257 @@
...
+ {
+	.aboutmecontainer {
+		width: 70%;
+		display: flex;
+		flex-direction: column;
+	}
+	
+}
+.abmimg {
+	transition: transform.2s;
```
Comment (mporam) [inline portfolio.css:166]: i dont think this works? atleast, it shouldnt. missing the space

File: portfolio.css [hunk 33]
```diff
@@ -0,0 +1,257 @@
...
+	width: 50px;
+	height: 50px;
+	position: relative;
+	opacity: 0.5;
+}
+.icon:hover {
+	opacity: 1;
+}
+.email:hover {
+	background-color:purple;
```
Comment (mporam) [inline portfolio.css:247]: youv used hex colors elsewhere, keep it consistent. dont use text based colors

**Pull request merged**